File: joao pedro/Problema0089/Problema0089.java
import java.util.Scanner;

public class Problema0089 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        double angulo = 0;

        System.out.println("Digite um angulo para ser classificado em agudo/reto/obtuso: ");
        angulo = Double.valueOf(input.nextLine());

        if (angulo == 90) {
            System.out.println("O angulo de " + angulo + " e: Reto");

        } else {
            if (angulo > 90) {
                System.out.println("O angulo de " + angulo + " e: Obtuso");

            } else {
                System.out.println("O angulo de " + angulo + " e: Agudo");

            }

        }

    }

}

File: joao pedro/Problema0089/Problema0089.js
const prompt = require("prompt-sync")({ sigint: true });

var angulo;

angulo=Number(prompt("Digite um angulo para ser classificado em agudo/reto/obtuso: "));

if (angulo == 90) {
    console.log("O angulo de "+angulo+" e: Reto");

} else {
    if (angulo > 90) {
        console.log("O angulo de "+angulo+" e: Obtuso");

    } else {
        console.log("O angulo de "+angulo+" e: Agudo");

    }

}